### PR TITLE
853 refactor meta strings to enums

### PR DESCRIPTION
**Pull request by raduiordache**
Fixes #<issue>

## Checklist:
- [x] My code follows the pykechain style
    - [x] I added type hinting to all functions
    - [x] I added documentation for all public functions
    - [x] I locally used `tox` to check for dists and docs errors
    - [x] My code passes the `pep8` and `flake8` linting checks and no warnings
    - [x] I removed unused imports, using `Code > Optimize Imports` in PyCharm
    - [ ] I used [`black`](https://github.com/psf/black) to format the new code
- [ ] I have added tests that prove my fix is effective or that my feature works
    - [ ] I committed fresh test cassettes
    - [ ] I have proper test coverage (don't decline the coverage of the test)
- [x] I asked another teammate to review the code
- [x] I update the Changelog.md with the appropriate changes.

### CODE REVIEW

File: pykechain/models/widgets/widgets_manager.py [hunk 1]
```diff
@@ -533,11 +534,12 @@ def add_attachmentviewer_widget(self,
...
-            "alignment": check_enum(alignment, Alignment, 'alignment'),
-            "imageFit": check_enum(image_fit, ImageFitValue, 'image_fit'),
-            "showDownloadButton": check_type(show_download_button, bool, 'show_download_button'),
-            "showFullscreenImageButton": check_type(show_full_screen_button, bool, 'show_full_screen_button'),
+            AssociatedObjectId.PROPERTY_INSTANCE_ID: attachment_property.id,
+            MetaWidget.ALIGNMENT: check_enum(alignment, Alignment, MetaWidget.ALIGNMENT),
+            MetaWidget.IMAGE_FIT: check_enum(image_fit, ImageFitValue, 'image_fit'),
+            MetaWidget.SHOW_DOWNLOAD_BUTTON: check_type(show_download_button, bool, 'show_download_button'),
+            MetaWidget.SHOW_FULL_SCREEN_IMAGE_BUTTON: check_type(show_full_screen_button, bool,
+                                                                 'show_full_screen_button'),
```
Comment (JelleBoersma) [inline pykechain/models/widgets/widgets_manager.py:542]: Remainders

File: pykechain/models/widgets/widgets_manager.py [hunk 2]
```diff
@@ -786,11 +793,11 @@ def add_service_widget(
...
-            "serviceId": service_id,
-            "emphasizeButton": emphasize_run,
-            "showDownloadLog": check_type(download_log, bool, "download_log"),
-            "showLog": True if download_log else check_type(show_log, bool, "show_log"),
-            "alignment": check_enum(alignment, Alignment, "alignment"),
+            AssociatedObjectId.SERVICE_ID: service_id,
+            MetaWidget.EMPHASIZE_BUTTON: emphasize_run,
+            MetaWidget.SHOW_DOWNLOAD_LOG: check_type(download_log, bool, "download_log"),
+            MetaWidget.SHOW_LOG: True if download_log else check_type(show_log, bool, "show_log"),
+            MetaWidget.ALIGNMENT: check_enum(alignment, Alignment, MetaWidget.ALIGNMENT),
```
Comment (JelleBoersma) [inline pykechain/models/widgets/widgets_manager.py:800]: The string in the final input for the `check____()` methods is the keyword of the Pykechain function/method, not the KE-chain meta, although they might overlap.

File: pykechain/models/widgets/widgets_manager.py [hunk 3]
```diff
@@ -1438,9 +1448,9 @@ def add_service_card_widget(
         service_id = check_base(service, Service, 'service', method=self._client.service)
 
         meta.update({
-            'serviceId': service_id,
-            'emphasizeButton': emphasize_run,
-            "alignment": check_enum(alignment, Alignment, "alignment"),
+            AssociatedObjectId.SERVICE_ID: service_id,
+            MetaWidget.EMPHASIZE_BUTTON: emphasize_run,
+            MetaWidget.ALIGNMENT: check_enum(alignment, Alignment, MetaWidget.ALIGNMENT),
```
Comment (JelleBoersma) [inline pykechain/models/widgets/widgets_manager.py:1453]: Same as above

**Review by JelleBoersma — CHANGES_REQUESTED** [review]
Few items to check, maybe on non-changed lines of code as well.

File: pykechain/models/widgets/widgets_manager.py (hunk 1 above)
Comment (JelleBoersma) [reply pykechain/models/widgets/widgets_manager.py:542]: Misread, discard this comment.

File: pykechain/models/widgets/widgets_manager.py (hunk 2 above)
Comment (raduiordache) [reply pykechain/models/widgets/widgets_manager.py]: Reverted bacl to 'alignment'

File: pykechain/models/widgets/widgets_manager.py (hunk 3 above)
Comment (raduiordache) [reply pykechain/models/widgets/widgets_manager.py]: Reverted back to 'alignment'

**Review by JelleBoersma — APPROVED** [review]

**Pull request merged**